src/components/TabBar/TabItems.tsx

```diff
@@ -0,0 +1,29 @@
+import {
```

Comment:
As this file isn't a component itself, the general convention is to use a non-capitalized letter in the beginning. `tabItems.tsx`